packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -102,14 +118,40 @@ abstract class RenderToggleable extends RenderConstrainedBox implements Semantic
     }
   }
 
+  /// Whether value of this control can be changed by user interaction.
+  ///
+  /// The control is considered interactive if the [onChanged] callback is
+  /// non-null. For example, a non-interactive checkbox is dislayed using a grey
+  /// color and its value cannot be changed.
   bool get isInteractive => onChanged != null;
 
+  /// The visual position of the control.
+  ///
+  /// When the control is inactive, the [value] is false and this animation has
+  /// the value 0.0. When the control is active, the value is [true] and this
+  /// animation has the value 1.0. When the control is changing from inactive
+  /// to active (or vice versa), [value] is the target value and this animation
+  /// gradually updates from 0.0 to 1.0 (or vice versa).
   CurvedAnimation get position => _position;
   CurvedAnimation _position;
 
+  /// Used to manipulate the visual position of the control.
```

Comment:
oh, it's for subclasses?

maybe make it clear this is for use by subclasses and not consumers of the class